source/AndroidXProject.cshtml

```diff
@@ -160,14 +160,26 @@
   {
     // Disabled as this build task doesn't really do anything anymore but we may use it again in the future
     <ItemGroup>
-      <None Include="..\..\source\buildtasks\annotation\bin\$(Configuration)\Xamarin.AndroidX.Annotation.BuildTasks.dll" Pack="True" PackagePath="build\$(TargetFramework)" />
+      <ProjectReference Include="..\..\source\buildtasks\annotation\Annotation-BuildTasks.csproj">
+        <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+        <OutputItemType>None</OutputItemType>
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+        <Pack>true</Pack>
+        <PackagePath>build\$(TargetFramework)</PackagePath>
+      </ProjectReference>
     </ItemGroup>
   }
 
   @if (@Model.NuGetPackageId == "Xamarin.AndroidX.VectorDrawable")
   {
     <ItemGroup>
-      <None Include="..\..\source\buildtasks\vectordrawable\bin\$(Configuration)\Xamarin.AndroidX.VectorDrawable.BuildTasks.dll" Pack="True" PackagePath="build\$(TargetFramework)" />
+      <ProjectReference Include="..\..\source\buildtasks\vectordrawable\VectorDrawable-BuildTasks.csproj">

```

Comment:
Instead of using a manual link, use a ProjectReference so that dependencies are built as well as making sure the correct .dll is used - the one that was just built.

Reply:
not anymore :)

Reply:
makes sense